android/src/main/java/io/ably/lib/push/ActivationContext.java

```diff
@@ -152,20 +151,15 @@ public static ActivationContext getActivationContext(Context applicationContext,
 
     protected void getRegistrationToken(final Callback<String> callback) {
         Log.v(TAG, "getRegistrationToken(): callback=" + callback);
-        FirebaseInstallations.getInstance().getToken(true)
-                .addOnCompleteListener(new OnCompleteListener<InstallationTokenResult>() {
-                    @Override
-                    public void onComplete(Task<InstallationTokenResult> task) {
-                        Log.v(TAG, "getRegistrationToken(): firebase called onComplete(): task=" + task);
-                        if(task.isSuccessful()) {
-                            /* Get new Instance ID token */
-                            String token = task.getResult().getToken();
-                            callback.onSuccess(token);
-                        } else {
-                            callback.onError(ErrorInfo.fromThrowable(task.getException()));
-                        }
-                    }
-                });
+        FirebaseMessaging.getInstance().getToken().addOnCompleteListener(task -> {
```

Comment:
I think a comment above this line to document the reason why you're not passing `true` to `getToken`.

Reply:
It's because it has zero parameters :) https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessaging#public-taskstring-gettoken

Reply:
ok, looks like I misread something.